modules/preview-react/radio/stories/radio.stories.mdx

```diff
@@ -0,0 +1,142 @@
+import {Alert} from './examples/Alert';
+import {Error} from './examples/Error';
+import {Inverse} from './examples/Inverse';
+import {LabelPosition} from './examples/LabelPosition';
+import {NoValue} from './examples/NoValue';
+import {RefForwarding} from './examples/RefForwarding';
+import {Disabled} from './examples/Disabled';
+import {Required} from './examples/Required';
+import {Custom} from './examples/Custom';
+import {StandaloneRadio} from './examples/StandaloneRadio';
+
+<Meta title="Preview/Inputs/Radio" component={RadioGroup} />
+
+# Canvas Kit Radio
+
+Radio Buttons allow a user to select one value from a predefined list of 7 or fewer options.
+
+[> Workday Design Reference](https://design.workday.com/components/inputs/radio-buttons)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic Example
+
+Our radio component includes a `RadioGroup` container and `RadioGroup.RadioButton`, which renders an
+individual radio button. Nest related `RadioGroup.RadioButton` buttons within a `RadioGroup` and
+provide a `name` prop to the `RadioGroup` to group the radio buttons together. Each
+`RadioGroup.Radio` must have a unique `value`. This value is used in conjunction with the `value`
+prop set on the `RadioGroup` to determine which radio button is selected. To tie it all together,
+provide an `onChange` handler to the `RadioGroup` to track the selected value as it changes.
+
+`RadioGroup` should be used in tandem with [Form Field](/components/inputs/form-field/) where the
+`useFieldset` prop is set to `true` to meet accessibility standards. This ensures the `label` text
+from `FormField` is attached to the `RadioGroup` and read out as a group for voiceover.
+
+<ExampleCodeBlock code={Basic} />
+
+Our example uses [React state](<(https://react.dev/learn/state-a-components-memory)>) to track the
+value of the `RadioGroup`.
+
+`RadioGroup` and `RadioGroup.Radio` support keyboard navigation through the proper use of WAI-ARIA
+[properties](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role#associated_wai-aria_roles_states_and_properties).
+
+### Error States
+
+Set the error prop of the wrapping `FormField` to `FormField.ErrorType.Alert` or
+`FormField.ErrorType.Error` to set the `RadioGroup` to the alert or error state, respectively. You
+will also need to set the `hintId` and `hintText` props on the `FormField` to meet accessibility
+standards. You must set an `aria-describedby` attribute on the `RadioGroup` element that matches the
+value of `hintId` set on the `FormField` element. These attributes ensure that the alert message is
+associated to the `RadioGroup` and read out by voiceover.
+
+#### Alert
+
+Use the alert state when a selection is valid but there is additional information.
+
+<ExampleCodeBlock code={Alert} />
+
+### Error
+
+Use the error state when the selection is no longer valid.
+
+<ExampleCodeBlock code={Error} />
```

Comment:
A couple things on the `Error States` section:

1. `### Error` should be `#### Error`
2. Can we move the entire `Error States` section so it's the last example (right above the `Component API` section)? This would be consistent with the rest of our input component MDXes which have `Error States` at the bottom.

Reply:
yep!